web/main/models.py

```diff
@@ -858,19 +849,8 @@ def refresh_search_index(cls):
         with connection.cursor() as cursor:
             try:
                 cursor.execute("REFRESH MATERIALIZED VIEW CONCURRENTLY internal_search_view")
-            except ProgrammingError as e:
```

Comment:
This doesn't actually work—for one thing the view name is `internal_search_view`, so the clause is never caught. If that's corrected, it _still_ doesn't work because the database is mid-transaction and can't commit the new indexing step. So I just removed this wrapped and put the "private" method at the top.